portfolio/src/main/webapp/script.js

```diff
@@ -103,3 +103,9 @@ async function getCommentCount(){
     commentContainer.innerHTML = commentHtml;
     
 }
+
+function deleteComments() {
+    fetch('/delete-data', {method: 'POST'});
+    window.location.reload();
```

Comment:
You can do this in a separate PR if you have time, but ideally you wouldn't need to reload the page to refresh the comments in the UI.

Reply:
Oh, okay. I did that because the changes did not reflect on the UI immediately, but the comments were deleted in the DataStore.

Reply:
I have implemented all the other issues raised in this PR.